guide/english/php/php-data-types/index.md

```diff
@@ -103,3 +103,23 @@ $herbie = new Car();
 echo $herbie->model;
 ?>
 ```
+
+### PHP Resource
+
+A resource is a special variable, holding a reference to an external resource. Resources are created and used by special functions. See the [appendix](http://php.net/manual/en/resource.php) for a listing of all these functions and the corresponding resource types. You can use [get_resource_type()](http://php.net/manual/en/function.get-resource-type.php) function to see resource type.
```

Comment:
here in the links, there is no appendix in the file being edited over here, so simply writing the word `appendix` will make it unclear to the reader about where the link will go.
Instead write `See the [official PHP documentation] for a ...` to make it more readable.